localization/beluga_demo_ndt_localization/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023-2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.5)
+project(beluga_demo_ndt_localization)
+
+find_package(ament_cmake REQUIRED)
+
+install(DIRECTORY config DESTINATION share/${PROJECT_NAME})
```

Comment:
you don't have a config directory right now.
You should have one though, maybe you forgot to push